src/hostnet_test/dune

```diff
@@ -2,5 +2,5 @@
  (name main)
  (libraries hostnet cmdliner alcotest logs.fmt protocol-9p mirage_dns
    mirage-clock-unix charrua-client.mirage forwarder sha
-   mirage-random-stdlib arp.mirage)
+   mirage-random-stdlib arp.mirage tcpip.ipv6 mirage-crypto-rng.unix)
```

Comment:
the mirage-random-stdlib is to be avoided (deprecated) and shouldn't be needed. Just use Mirage_crypto_rng directly (and Mirage_crypto_rng_unix.use_default ()).